ios-trivia-game/ViewControllers/ProfileViewController.swift

```diff
@@ -7,13 +7,47 @@
 //
 
 import UIKit
+import AFNetworking
 
 class ProfileViewController: UIViewController {
 
+    @IBOutlet weak var contentViewCenterY: NSLayoutConstraint!
+    @IBOutlet weak var contentView: UIView!
+    @IBOutlet weak var profileImageView: UIImageView!
+    @IBOutlet weak var nicknameLabel: UILabel!
+    @IBOutlet weak var nameLabel: UILabel!
+    @IBOutlet weak var emailLabel: UILabel!
+    
+    fileprivate var user: User!
+    
+    override func viewWillAppear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        // self.contentViewCenterY.constant += self.view.bounds.height
+        self.contentView.center.x += self.contentView.frame.width
+        self.contentView.layoutIfNeeded()
+        self.profileImageView.alpha = 0.0
+        
+        UIView.animate(withDuration: 0.6, animations: {
+            // self.contentViewCenterY.constant -= self.view.bounds.height
+            
+            self.contentView.center.x -= self.contentView.frame.width
+            self.profileImageView.alpha = 1
+            self.contentView.layoutIfNeeded()
+        })
+    }
+    
     override func viewDidLoad() {
         super.viewDidLoad()
-
-        // Do any additional setup after loading the view.
+        
+        self.user = User.currentUser
+        
+        self.profileImageView.clipsToBounds = true
+        self.profileImageView.setImageWith(URL(string: self.user.photoUrl!)!)
+        self.profileImageView.layer.cornerRadius = self.profileImageView.frame.size.width / 2;
+        
+        self.nicknameLabel.text = "somenickname"
```

Comment:
I'm thinking besides pulling data from FB, we might want to append to the profile and store it in firebase to complement what fb gives us.  Nickname for example